app/components/post_body/post_body.js

```diff
@@ -507,16 +507,16 @@ export default class PostBody extends PureComponent {
                         {body}
                     </View>
                     {isFailed &&
-                    <TouchableOpacity
-                        onPress={onFailedPostPress}
-                        style={style.retry}
-                    >
-                        <Icon
-                            name='ios-information-circle-outline'
-                            size={26}
-                            color={theme.errorTextColor}
-                        />
-                    </TouchableOpacity>
+                        <TouchableOpacity
```

Comment:
Good to know I'm not the only one who's ESLint complains about these. I wish I knew why Jenkins doesn't report these